test/test_extended_models.py

```diff
@@ -196,24 +212,49 @@ def test_schema_meta_validation(model_fn):
         if missing_fields or unsupported_fields:
             problematic_weights[w] = {"missing": missing_fields, "unsupported": unsupported_fields}
         if w == weights_enum.DEFAULT:
+
             if module_name == "quantization":
                 # parameters() count doesn't work well with quantization, so we check against the non-quantized
                 unquantized_w = w.meta.get("unquantized")
                 if unquantized_w is not None and w.meta.get("num_params") != unquantized_w.meta.get("num_params"):
                     incorrect_params.append(w)
+
+                # the methodology for quantized ops count doesn't work as well, so we take unquantized FLOPs instead
+                calculated_ops = get_ops(model=None, module_name="models", model_name=model_name, weight=unquantized_w)
```

Comment:
We don't have to do this estimation. We can follow the same approach as with the `num_params`. More precisely:
we fetch the `unquantized_w.meta.get("_ops")` and confirm that the match what we have here. Basically we reproduce the logic on lines 219-220.